### DISCUSSION TITLE
Keyball44_via_v1.2.0をFlash後にトラックボールのみ反応しない

### DISCUSSION THREAD

**Discussion by Hakuchu1Obaga** [Yowkees/keyball#398] [Q&A] [answered]
Keyball44をQMK Firmware 0.22.2に対応のためKeyball44_via_v1.2.0をFlashしたのですが、トラックボールのみ反応しなくなりました。QMK0.18の時には正常に反応していたのですが。手順としては、添付の状態のままFlashしました。Remap上はlatest firmwareのリンク先で前に保存したマップをFlashできました。
トラックボールのみ動かない原因がわかりません。よろしくお願いいたします。

![Flash Firmware1](https://github.com/Yowkees/keyball/assets/121614257/f2ee53a4-0b25-490f-9c03-dc9526ac1680)

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。

バージョンアップによりトラックボールが動かなくなることは考えにくいので、
おそらくタイミングよく、潜在していた基板上の問題が表れてしまったものと思われます。

電源を落としたうえで、トラックボール基板の抜き差しを試していただけますでしょうか。
また、その際に下記画像のように7Pコンスルーピンの変形がないかもご確認ください。変形がある場合はピンセットで成形していただくと導通しやすいと思います。

以上、お手数ですがご確認お願いいたします。

![2_conthru_henkei](https://github.com/Yowkees/keyball/assets/84209772/3e2871c0-2405-462d-9143-12983e928778)

**Reply by Hakuchu1Obaga** [reply, marked as answer]
何度もすみません。
7Pコンスルーピンの変形確認、トラックボール基板の抜き差しを行い外観上異常は見当たりませんでしたが、依然トラックボールの反応がありません。相談ですが、こちらのハードウェアを送付し、診断を頂けることは可能でしょうか？
可能であれば、別途ご案内頂きたく存じます。
ご検討のほどよろしくお願いいたします。
![P_20231024_202630](https://github.com/Yowkees/keyball/assets/121614257/b188f5af-b5d0-4971-8de3-92b3b644446f)

**Reply by Yowkees** [reply]
了解しました。
念のため、トラックボール基板やジャンパー、ProMicroの半田付けを再確認をお願いします。
もし解決しないようでしたら白銀ラボウェブショップのお問合せよりご連絡ください。その後の修理方法をご相談させていただきます。

以上、よろしくお願いいたします。

**Reply by ruemura3** [reply]
同じような状況で、最新版のRemapを使うためにKeyball44に対してKeyball44_via_v1.3.2をFlashしたのですが、トラックボールのみ反応しなってしまいました。
左側キーボードにFlashを行った後、トラックボールが反応しなくなりましたが、右側キーボードは元のままなので右側をPCと接続して使えばトラックボールは反応する状況です。
解決策をご教授いただけますと幸いです。

**Comment by Hakuchu1Obaga** [comment]
承知しました。
バージョンアップ手順に間違いないのでしたらそちらは安心しました。
基盤の接続の方確認します。
ご回答のほどありがとうございました。

**Comment by Hakuchu1Obaga** [comment]
ご検討を頂きありがとうございます
もう一度再確認して相談をさせて頂きます
よろしくお願いします

2023年10月26日(木) 8:22 Yowkees ***@***.***>:
